xbmc/DynamicDll.cpp

```diff
@@ -25,17 +25,19 @@
 
 using namespace XFILE;
 
-DllDynamic::DllDynamic()
+DllDynamic::DllDynamic():
```

Comment:
this is completely pointless comment but moving the `:` to the next row gives a nice symmetry as everything lines up :)